### ISSUE TITLE
RatioResize not compliant with TensorBBox and TensorPoint

### ISSUE THREAD

**Issue by antoinebon** [fastai/fastai2#285]
```python
class RatioResize(Transform):
    'Resizes the biggest dimension of an image to `max_sz` maintaining the aspect ratio'
    def __init__(self, max_sz, resamples=(Image.BILINEAR, Image.NEAREST)):
        self.max_sz,self.resamples = max_sz,resamples

    def encodes(self, x:(Image.Image,TensorBBox,TensorPoint)):
        w,h = x.size
        if w >= h: nw,nh = self.max_sz,h*self.max_sz/w
        else:      nw,nh = w*self.max_sz/h,self.max_sz
        return Resize(size=(int(nh),int(nw)), resamples=self.resamples)(x)
```

Problem in `RatioResize.encodes`: `TensorBBox` and `TensorPoint` don't have `size` attribute, thus `x.size` will fail

**Comment by antoinebon** [comment]
This seems to do the trick:
```python
class RatioResize(Transform):
    order=1
    'Resizes the biggest dimension of an image to `max_sz` maintaining the aspect ratio'
    def __init__(self, max_sz, resamples=(Image.BILINEAR, Image.NEAREST)):
        self.max_sz,self.resamples = max_sz,resamples

    def encodes(self, x:(Image.Image,TensorBBox,TensorPoint)):
        w,h = _get_sz(x)
        if w >= h: nw,nh = self.max_sz,h*self.max_sz/w
        else:      nw,nh = w*self.max_sz/h,self.max_sz
        return Resize(size=(int(nh),int(nw)), resamples=self.resamples)(x)
```

**Comment by sgugger** [comment]
Yes, this was an internal contribution and I didn't catch that. Thanks for flagging it! Would you mind making a PR with the fix?

**Comment by antoinebon** [comment]
Sure, I'll try to get a hang of the new dev environment first :-)

**Comment by sgugger** [comment]
Closed by #288
